test/extended/prometheus/prometheus.go

```diff
@@ -76,6 +78,27 @@ var _ = g.Describe("[Feature:Prometheus][Late] Alerts", func() {
 
 		e2e.Logf("Watchdog alert is firing")
 	})
+
+	g.It("shouldn't exeeced the 500 series limit of total series sent via telemetry from each cluster", func() {
```

Comment:
s/exeeced/exceed/

Reply:
Spelled it correctly in the commit lol 🤦‍♀